resources/lib/screensaver.py

```diff
@@ -116,8 +116,12 @@ def get_images(self, override=False):
         # Read google images from json file
         self.images = []
         if kodiutils.get_setting_as_int("screensaver-mode") == 0 or kodiutils.get_setting_as_int("screensaver-mode") == 2 or override:
-            with open(IMAGE_FILE, "r", encoding="utf-8") as f:
-                images = f.read()
+            try:
```

Comment:
Can you ifdef using the PY3 "global" instead of the try except? It will be easier to track and drop in the future, when py2 support is removed

Reply:
What / where is the PY3 global? I can't find it anywhere in your code.

Reply:
You're right, the addon was ported to python3 by rechi not me
Thanks